### PR TITLE
Setting up playstv

### PR DESCRIPTION
**Pull request by burnNturn**

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -23,10 +23,15 @@ def update
     end
   end
   
+  def videos
+    plays_tv = PlaysTv.new
+    @response = PlaysTv.get_videos('burnNturn3')
```
Comment (Jabronious) [inline app/controllers/users_controller.rb]: Do not want to have this hardcoded for you everytime! :)

File: config/routes.rb [hunk 2]
```diff
@@ -12,10 +12,12 @@
   resources :users, only: [:show, :update]
   # The priority is based upon order of creation: first created -> highest priority.
   # See how all your routes lay out with "rake routes".
-
+  get '/videos' => 'users#videos', as: '/videos'
   # You can have the root of your site routed with "root"
   # root 'welcome#index'
   root 'home#index'
+  
+  #get 'playstv', to: 'home#playstv'
```
Comment (Jabronious) [inline config/routes.rb]: Do we need this route or is it suppose to be commented out?

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (burnNturn) [reply app/controllers/users_controller.rb]: opps, thought I changed that already. lol

File: config/routes.rb (hunk 2 above)
Comment (burnNturn) [reply config/routes.rb]: It is working with out it, but I am not sure if we may need it in the futrue, so I thought I would leave commented out for now.

Comment (Jabronious) [reply config/routes.rb]: 👍

**Pull request merged**